src/main/java/com/cloudbees/jenkins/GitHubRepositoryName.java

```diff
@@ -118,7 +119,10 @@ protected GHRepository adapt(GitHub item) {
      *
      * This is useful if the caller only relies on the read access to the repository and doesn't need to
      * walk possible candidates.
+     *
+     * Result can be null
```

Comment:
remove this line, checkfornull is enough or if you want java doc then change to `@return desc`